src/Adliance.AzureTools/CopyDatabase/Parameters/CopyDatabaseParameters.cs

```diff
@@ -5,8 +5,8 @@ namespace Adliance.AzureTools.CopyDatabase.Parameters
     [Verb("copy-database", HelpText = "Copies an existing database (structure and data) to another one. If the target database already exists, it will be overwritten. Please note that this uses SqlPackage and currently only works on Windows.")]
     public class CopyDatabaseParameters
     {
-        [Option('s', "source", Required = true, Default = "", HelpText = "The connection string to the source database.")] public string Source { get; set; } = "";
-        [Option('t', "target", Required = true, Default = "", HelpText = "The connection string to the target database.")] public string Target { get; set; } = "";
+        [Option("source", Required = true, Default = "", HelpText = "The connection string to the source database.")] public string Source { get; set; } = "";
```

Comment:
Are we sure about that? I think the warning is enough, I'm not happy to enforce a "blown up" command line interface?